content/en/tracing/profiler/getting_started.md

```diff
@@ -256,6 +256,88 @@ The Datadog Profiler requires Go 1.12+. To begin profiling applications:
 [6]: /tracing/visualization/#services
 [7]: /tracing/guide/setting_primary_tags_to_scope/#environment
 {{< /programming-lang >}}
+{{< programming-lang lang="ruby" >}}
+
+<div class="alert alert-warning">
+Datadog Ruby Profiler is currently in public beta. We strongly advise evaluating the profiler in a non-sensitive environment before deploying in production.
+</div>
+
+The Datadog Profiler requires MRI Ruby 2.1+ and **CPU-time profiles are only available on Linux platforms**. To begin profiling applications:
+
+1. If you are already using Datadog, upgrade your agent to version [7.20.2][1]+ or [6.20.2][1]+.
+
+2. Add the `ddtrace` and `google-protobuf` gems to your `Gemfile` or `gems.rb` file:
+
+    ```ruby
+    gem 'ddtrace'
+    gem 'google-protobuf', '~> 3.0'
+    ```
+
+    **Note**: Profiler is available in the `ddtrace` library for versions 0.49+.
+
+2. Install the gems with `bundle install`
+
+3. You can auto-enable the profiler by setting the `DD_PROFILING_ENABLED` environment variable to `true`:
```

Comment:
IMO bullet 3 should be structured something like

> 3. You can configure the profiler with environment variables:
> ```sh
> DD_PROFILING_ENABLED=true
> DD_ENV=prod
> DD_SERVICE=my-web-app
> DD_VERSION=1.0.3
> ```
> or in code:
> ```ruby
> Datadog.configure do |c|
>    ...
> end
> ```
> In a Rails application, you could put the code config block in `config/initializers/datadog.rb`.

Note: no backslashes after env vars since this isn't a full command; they make sense to have if we put the command to run in the same code block.